# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1265018165 -3600
#      Mon Feb 01 10:56:05 2010 +0100
# Node ID c4c2bda71f82a7ae92bd0728668f686ea42e67ae
# Parent  cb7d1b220f1cc029ec08a9987e116665654b90e5
fix typo in strncmpi docstring

diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,12 @@
+2010-02-01  Jaroslav Hajek  <highegg@gmail.com>
+
+	* strfns.cc (Fstrncmpi): Fix typo in docstring.
+
 2010-01-30  Jaroslav Hajek  <highegg@gmail.com>
 
 	* ov-list.h, ov-list.cc, OPERATORS/op-list.cc: Remove.
 
 	* Makefile.am, OPERATORS/module.mk: Update.
 	* debug.cc: Remove #include.
 	* ov-base.cc: Ditto.
 	* ov-cs-list.h: Ditto.
diff --git a/src/strfns.cc b/src/strfns.cc
--- a/src/strfns.cc
+++ b/src/strfns.cc
@@ -783,17 +783,17 @@ strncmpi_str_op (const std::string& s1, 
   octave_idx_type l1 = s1.length (), l2 = s2.length ();
   return (n > 0 && n <= l1 && n <= l2 
           && std::equal (s1.data (), s1.data () + n, s2.data (),
                          icmp_char_eq ()));
 }
 
 DEFUN (strncmpi, args, ,
   "-*- texinfo -*-\n\
-@deftypefn {Built-in Function} {} strncmp (@var{s1}, @var{s2}, @var{n})\n\
+@deftypefn {Built-in Function} {} strncmpi (@var{s1}, @var{s2}, @var{n})\n\
 Returns 1 if the first @var{n} character of @var{s1} and @var{s2} are the same,\n\
 disregarding case of alphabetic characters, and 0 otherwise.\n\
 \n\
 If either @var{s1} or @var{s2} is a cell array of strings, then an array\n\
 of the same size is returned, containing the values described above for\n\
 every member of the cell array.  The other argument may also be a cell\n\
 array of strings (of the same size or with only one element), char matrix\n\
 or character string.\n\
